runtime/src/main/java/org/corfudb/protocols/wireprotocol/MultipleReadRequest.java

```diff
@@ -0,0 +1,38 @@
+package org.corfudb.protocols.wireprotocol;
+
+import io.netty.buffer.ByteBuf;
+
+import java.util.Arrays;
+import java.util.List;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.Getter;
+
+/**
+ * Created by maithem on 7/28/17.
+ */
+@Data
+@AllArgsConstructor
+public class MultipleReadRequest implements ICorfuPayload<MultipleReadRequest> {
+    @Getter
+    final List<Long> addresses;
```

Comment:
The other option would be a set, but that would impose some needless performance penalty and memory overhead. My opinion here is to go with the set, and if for some reason the caller uses the same address multiple times, we will return it multiple times. Besides, at the protocol level it is impractical to enforce a set...